frontend/src/main/components/Leaderboard/LeaderboardTable.js

```diff
@@ -18,12 +18,9 @@ export default function LeaderboardTable({ leaderboardUsers }) {
         },
         {
             Header: "Total Wealth",
-            id: "totalWealth",
-            accessor: (row, _rowIndex) => {
-                return USD.format(row.totalWealth);
-            },
+            accessor: "totalWealth",
             Cell: (props) => {
-                return <div style={{ textAlign: "right" }}>{props.value}</div>;
+                return <div style={{ textAlign: "right" }}>{(USD.format(props.value))}</div>;
             },
         },
         {
```

Comment:
Why is this change in this PR?  It is unrelated to adding the Students Database Table.

Could you revert this file to the original so that this is not cluttering up the PR?  It would be mergeable sooner with this included.